flowdb/Pipfile

```diff
@@ -8,6 +8,7 @@ psutil = "*"
 
 [dev-packages]
 black = {extras = ["jupyter"],version = "==24.2.0"}
+jinja2 = "*"
```

Comment:
Should go in `flowdb/testdata/test_data` rather than the root one, isn't needed in the base image, only the test/synth ones.